### PR TITLE
Split deployments.json by network

### PR DESCRIPTION
**Pull request by MaciejNadolski98**

### CODE REVIEW

**Review by yuchenlintt — CHANGES_REQUESTED** [review]
Thanks for the change! I have some preliminary comments attached to this review.

Since this file is such an important piece of our repo, I also recommend we have 3 engineers do a full diff check of the final commit of this PR to confirm all the addresses are identical.

Could you preserve the git blame version history in `deployments-mainnet.json` instead of in `deployments-ropsten.json`? You should be able to achieve this by rewriting this branch in the following sequence:
1. git mv deployments.json deployments-mainnet.json
2. git add, commit, etc.
3. Cut ropsten addresses and paste to deployments-ropsten.json
4. Cut unknown addresses and paste to deployments-unknown.json

File: deployments-ropsten.json [hunk 1]
```diff
@@ -27,8 +9,8 @@
       "address": "0x1cB0906955623920c86A3963593a02a405Bb97fC"
     },
     "testTrustToken": {
-      "txHash": "0xb48866f845db4248cf9413f41587c546acd896ffa680cf9b8364457adff49a5b",
-      "address": "0x90c33ef2CfDc90266997b84a8EbaC616B1F89C2b"
+      "txHash": "0xa646bc5731878a7dbeed46b51b991101ee1fe2bbc7186333c7c4968da525901d",
```
[the remark this answers is not in the document]
Comment (yuchenlintt) [reply deployments-ropsten.json]: It looks like this is fixed now. Resolving this comment.

File: deployments-mainnet.json [hunk 2]
```diff
@@ -1,312 +1,4 @@
 {
```
Comment (yuchenlintt) [inline deployments-mainnet.json:1]: This file was the most stressful to review, but now it's the easiest! LGTM.

File: deployments-ropsten.json [hunk 3]
```diff
@@ -0,0 +1,292 @@
+{
```
Comment (yuchenlintt) [inline deployments-ropsten.json:1]: I'm confirming I diffed this file against the ropsten section of the original deployments.json and found no changes.

File: deployments-unknown.json [hunk 4]
```diff
@@ -0,0 +1,20 @@
+{
```
Comment (yuchenlintt) [inline deployments-unknown.json:1]: I'm confirming I diffed this file against the unknown section of the original deployments.json and found no changes.

**Review by yuchenlintt — CHANGES_REQUESTED** [review]
One more change in marsDeploy.sh, otherwise this PR LGTM!

**Review by yuchenlintt — APPROVED** [review]

**Review by barrutko — APPROVED** [review]

**Review by MiksuJak — APPROVED** [review]

**Pull request merged**